test/e2e/adminapi.go

```diff
@@ -64,7 +66,7 @@ func adminRequest(ctx context.Context, method, path string, params url.Values, i
 		return nil, err
 	}
 	defer func() {
-		_, _ = resp.Body.Read(nil)
+		adminResponseBody, _ = ioutil.ReadAll(resp.Body) //ioutil.ReadAll(resp.Body)
```

Comment:
1. Why do we need to write into a global var?
1. Are we going to do anything with `adminResponseBody` later in this test?

Reply:
Yes, We require this global variable to extract response body so that the logs can be matched to verify and pass this e2e test ...